packages/blade/tsconfig-generate-types.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "./tsconfig.json",
+  "exclude": ["**/**.test.ts", "**/**.test.tsx", "__tests__"],
+  "include": ["src", "src/tokens/theme/theme.d.ts"],
+  "compilerOptions": {
+    "declaration": true,
+    "emitDeclarationOnly": true,
+    "noEmit": false,
+    "declarationDir": "build/types"
```

Comment:
Can add a brief comment around values for these options. They way it's mentioned in RFC Phase 2.

Reply:
It's ts stuff so you get documentation out of the box once you hover over it 😄 . Anyways added the comments